src/containers/reviews/reviewsListContainer.js

```diff
@@ -15,6 +15,9 @@ const mapDispatchToProps = (dispatch) => {
     getData: (id) => {
```

Comment:
I like the way you containerized this reviewsList. The dispatched functions look like they cover a lot of functionality.